### PR TITLE
Create .gitattributes

### PR DESCRIPTION
**Pull request by reedy**
Ported from https://github.com/JakubOnderka/PHP-Console-Highlighter/pull/18

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1,7 @@
+/examples export-ignore
```
Comment (jrfnl) [inline .gitattributes:1]: Did I miss this earlier or did you just add `/examples` to the list as well ?

Examples should probably be considered documentation, similar to the `README` and would, in that case, normally be included in a distribution.

Comment (reedy) [reply .gitattributes:1]: No it was already there

Comment (jrfnl) [reply .gitattributes:1]: :grin: In that case, I haven't said anything...

**Comment by reedy** [comment]
@grogy Want to merge this?

**Comment by reedy** [comment]
https://github.com/php-parallel-lint/PHP-Console-Highlighter/pull/6 is needed first

**Comment by grogy** [comment]
Good job! Thank you

**Pull request merged**